docs/pages/docs/providers/elixir.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Elixir
+---
+
+# {% $markdoc.frontmatter.title %}
+
+Elixir is detected if a `mix.exs` file is found.
```

Comment:
I think we should mention that Elixir with a DB is not currently supported

Reply:
A note at the bottom would be good. That said, if someone comes with a broken Elixir with database we can fix it for them ASAP

Maybe a note about reaching out on Discord?

Reply:
I've just created a PR to add Ecto & Postgress support to Elixir
https://github.com/railwayapp/nixpacks/pull/485

Will merge current PR for now and can add this bottom line if we the other PR took longer to merge